web/src/main/java/org/cbioportal/web/util/ClinicalDataIntervalFilterApplier.java

```diff
@@ -134,7 +138,7 @@ else if (value.startsWith(gt)) {
 
         // TODO also add startInclusive and endInclusive as a filterValue parameter?
```

Comment:
Is addressing this TODO within the scope of this PR?

Reply:
No. And I have no idea about this

Reply:
I guess we never needed to customize `startInclusive` and `endInclusive` options so far. Feel free to remove the TODO.

Reply:
Sure, I'll clean it up